networkx/conftest.py

```diff
@@ -238,6 +239,7 @@ def add_nx(doctest_namespace):
     "algorithms/bipartite/spectral.py",
     "algorithms/centrality/current_flow_betweenness.py",
     "algorithms/centrality/current_flow_betweenness_subset.py",
+    "algorithms/centrality/domirank.py",
```

Comment:
If we move the scipy import to the `case "analytical` block, we should be able to remove this requirement.